### ISSUE TITLE
Get confidence intervals for survival metrics in mlr3proba

### ISSUE THREAD

**Issue by Selenia94** [mlr-org/mlr3proba#271]
Hello :) 

First of all thanks for your amazing work on mlr3proba ! :)

It may be an open question so I am not sure if it is the right channel for this, but I was wondering what could be the best way to get confidence intervals for survival metrics in mlr3proba (e.g for the C-index) ?

Would it make sense, for instance, to bootstrap the test set and repeat the computation of survival metrics on each bootstrap sample ? In such a case, would the model need to be entirely re-tuned for each bootstrap sample ?

Thank you in advance !
Have a nice day !

Selenia

**Comment by RaphaelS1** [comment]
Hiya, unfortunately it is an open question! There's a lot of papers talking about how (not) to get confidence intervals around C-index. The simplest way to get some sort of range is just to use k-fold resampling and plot the results in each fold. This is not a confidence interval and nor does it provide any generalisation guarantees but it does at least drive towards a useful quantity. 

Alternatively you could benchmark across many (maybe 30+) independent tasks and calculate a mean and standard error across these. 

Otherwise I guess bootstrapping as you suggest. About tuning you would generally only tune once on training data before predicting so I don't think you'd need to retune.

**Comment by RaphaelS1** [comment]
> Do you mean use k-fold resampling on the whole dataset instead of train/test splitting ?

It's a bit of a tricky topic. The usual thing is to first split your data into data to tune the model on and then data to evaluate the model with. You tune the model to find optimal parameters on the tuning data then you evaluate with K-fold CV on the testing data.

> I am not sure to understand, do you mean creating independent tasks with different "samples" of the same dataset ?

No, completely independent data.

**Comment by Selenia94** [comment]
Ok it seems all clear to me ! 
Again thank you a lot for your reply :)
